dist/react-filterbar.js

```diff
@@ -7767,9 +7767,11 @@ var FilterBarActor = exports.FilterBarActor = (function () {
         var buttons = filterBarStore.quickFilters[blockName];
         Object.keys(buttons).map(function (buttonName) {
           var filters = filterBarStore.quickFilters[blockName][buttonName].filters;
-          Object.keys(filters).map(function (filterName) {
-            self.disableFilter(filters[filterName].filter);
-          });
+          if (typeof filters == "object") {
```

Comment:
Actually, while it would guard against `null`, it's primarily to avoid attempting to iterate on strings — [author block labels in particular](https://github.com/jobready/react-filterbar/pull/48/files#diff-04c6e90faac2675aa89e2176d2eec7d8R131).